x/authority/types/client_ctx.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+)
+
+var (
+	ClientCtx client.Context
+)
+
+func RegisterClientCtx(clientCtx client.Context) error {
```

Comment:
should we remove the return type `error` as it always succeed

Reply:
We actually don't need this file